source/Mlos.Python/mlos/Optimizers/ExperimentDesigner/UtilityFunctions/UtilityFunction.py

```diff
@@ -13,3 +19,24 @@ class UtilityFunction(ABC):
     @abstractmethod
     def __call__(self, feature_values_pandas_frame):
         raise NotImplementedError("All subclasses must implement this.")
+
+
+class GreedyUtilityFunction(UtilityFunction):

```

Comment:
There are many ways to be greedy. Could we name it PredictedValueUtilityFunction?

Reply:
Also, could we have it in its own file please?

Reply:
there's also many ways to use the predicted value ;) But sure, I'll rename / move.